projects/pgai/pgai/vectorizer/embedders/openai.py

```diff
@@ -35,6 +36,18 @@
 )
 
 
+class ResponseWithRead:
+    def __init__(self, response: "AsyncAPIResponse[types.CreateEmbeddingResponse]"):
+        default_ijson_buffer = 64 * 1024
```

Comment:
Can we extract this to a constant and also pass that value as the `buf_size` parameter to ijson.async_parse? I'm concerned that we're implicitly relying on them being the same.

Reply:
Good one 👍🏼